integration-tests/api-gateway/contract/routes/CredentialIssueRouter.ts

```diff
@@ -0,0 +1,55 @@
+import express from "express";
+import asyncify from "express-asyncify";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { stackOutputs } from "../../../resources/cloudformation-helper";
+import { environment } from "../../env-variables";
+import { formatJwtForPactTest } from "../utils/PactJwtFormatter";
+import { IncomingHttpHeaders } from "http";
+
+const logger = new Logger({
+  logLevel: "DEBUG",
+  serviceName: "CredentialIssueRouter",
+});
+
+let publicAPI: string;
+
+export const credentialIssueRouter = asyncify(express.Router());
+credentialIssueRouter.post("/", async (req, res) => {
+  logger.debug(`Recieved Request - Type: ${req.method}`);
+
+  const passedThroughHeaders = convertExpressHeadersToFetchHeaders(req.headers);
+
+  if (!publicAPI) {
+    const preOutput = await stackOutputs(process.env.STACK_NAME);
+    publicAPI = `${preOutput.PublicApiGatewayId}`;
+  }
+
+  const response = await fetch(
+    `https://${publicAPI}.execute-api.eu-west-2.amazonaws.com/${environment}/credential/issue`,
```

Comment:
Ok interesting! So we are running a local server that points to the deployed endpoints?